types/src/transaction/automated_transaction.rs

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Aptos Foundation
```

Comment:
Switch copyright line order to make in reverse chronological order of contribution.